src/server/cluster/incoming_slot_migration.h

```diff
@@ -47,13 +47,16 @@ class IncomingSlotMigration {
     return source_id_;
   }
 
+  size_t GetKeysNumber() const;
```

Comment:
How about:
```suggestion
  size_t GetKeyCount() const;
```
?

Reply:
It doesn't matter for me, if I do update of the PR I will apply this